pipeline_utils/schemas/yaml_file_format.py

```diff
@@ -0,0 +1,41 @@
+yaml_file_format_schema = {
+    ## Schema #########################
+    '$schema': 'https://json-schema.org/draft/2020-12/schema',
+    '$id': '/schemas/YAMLFileFormat',
+    'title': 'YAMLFileFormat',
+    'description': 'Schema to validate a YAML description of a FileFormat',
+    'type': 'object',
+    'properties': {
+        'name': {
+            'description': 'Name of the FileFormat',
+            'type': 'string'
+        },
+        'description': {
+            'description': 'Description of the FileFormat',
+            'type': 'string'
+        },
+        'extension': {
+            'description': 'Extension of the FileFormat',
+            'type': 'string'
+        },
+        'file_types': {
+            'description': 'File types that can use the FileFormat',
+            'type': 'array',
+            'items': {
+                'type': 'string',
+                'pattern': 'FileReference|FileProcessed'
```

Comment:
Should include `FileSubmitted` now as well
```suggestion
                'pattern': 'FileReference|FileProcessed|FileSubmitted'
```

Reply:
I totally forgot about FileSubmitted, added